DOC-295 | Clarifications - Global Write Transaction Lock (#1273)
arangodb/docs: 3.10/backup-restore.md

- **The Global Write Transaction Lock**

  To create a consistent snapshot of an ArangoDB single server or
  cluster deployment, all transactions need to be suspended in order for the
  state of a deployment to be consistent. However, there is no way for ArangoDB
  to know by its own when this time comes. This is why a hot backup needs to
  aquire a global write transaction lock in order to create the backup in a
  consistent state.

>   On a single server instance, this lock is eventually obtained rather quickly

Reviewer: "eventually ... rather quickly" seems like a contradiction

  and the hot backup is created within a very short amount of time.

  However, in a cluster, this process is more complex. One Coordinator tries to
  obtain the global write transaction lock on all _DB-Servers_ simultaneously.

>   Depending on the activity on the cluster, this could take some time for the

Reviewer: in or of
```suggestion
  Depending on the activity in the cluster, it can take some time for the
```

  Coordinator to acquire all the locks the cluster needs. Grabbing all the

>   necessary locks at once might not be always successful, leading to apparent

Reviewer: ```suggestion
  necessary locks at once might not always be successful, leading to apparent
```

>   dead times as if the cluster's write operations are suspended.

Reviewer: "dead times" seems like a poor choice of words. Is there an alternative for "dead"?

  This process can happen multiple times until all locks are obtained.

>   The operator has control over the length of the time during which the lock

Reviewer: "The operator" is ambiguous. It could refer to our Kubernetes operator as well as to the person administrating the deployment.
